src/components/Screen.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+
+const Screen = props => {
+  const {
+    value: {
+      next,
+      total,
+    },
+  } = props;
+
+  return (
+    <div className="Display">
+      <span className="Screen">{next || total || '0'}</span>
+    </div>
+  );
+};
+
```

Comment:
[Optional]

- The app is looking great and working fine, but when it comes to User experience, it's not that good. Currently, I am not able to see which operation I choose on the screen. Ex: If I try to do `4 + 5 `, then I won't able to see the `+` sign on the screen. Would you mind changing it up a bit (in terms of the display), a user to see the digits (and, hopefully, operations that get typed for great User experience?

Reply:
Thank you for the review. For sure I'll implement the optional changes.